cc/mcts_tree.cc

```diff
@@ -302,13 +351,13 @@ MctsNode* MctsTree::SelectLeaf(bool allow_pass) {
       return node;
     }
 
-    auto child_action_score = node->CalculateChildActionScore();
-    absl::Span<const float> allowed_moves(child_action_score);
+    PaddedArray<float, kNumMoves> child_action_score;
+    node->CalculateChildActionScoreSse(child_action_score);
```

Comment:
does this need to be extracted to "SelectLeafSse"?  Should we have mcts_tree_sse.cc?  where does the madness stop?

Reply:
As long as we only have a couple of these methods, probably the most pragmatic solution for non-x86 platforms is to add ifdefs inside of the *Sse methods that call the non-SSE implementations.